Source/Atomic/Resource/Image.cpp

```diff
@@ -1211,6 +1216,91 @@ bool Image::SaveJPG(const String& fileName, int quality) const
         return false;
 }
 
+bool Image::SaveDDS(const String& fileName) const
+{

```

Comment:
What about jpg, png and tga saving? Would that not make sense in tooling as well?